scripts/bundles/cli.ts

```diff
@@ -45,6 +45,28 @@ export async function cli(opts: BuildOptions): Promise<ReadonlyArray<RollupOptio
   dts = dts.replace('@stencil/core/internal', '../internal/index');
   await fs.writeFile(join(opts.output.cliDir, dtsFilename), dts);
 
+  // bundle config-flags.js
+  const esConfigFlagsOutput: OutputOptions = {
+    format: 'es',
+    file: join(outputDir, 'config-flags.js'),
+    preferConst: true,
+    sourcemap: true,
+    banner: getBanner(opts, `Stencil CLI Config Flags`, true),
+  };
```

Comment:
this is my first time messing around in these bundle files, so I'm not sure this is the best approach, that we want to do this, etc.

There is also some slightly strange stuff going on here. For instance, for the `cli` module in general, we currently do not bundle the `index.d.ts` file which is generated by Typescript during compilation, but instead we read file `src/cli/public.d.ts` from disk, replace a string import with a different one to make the path to the stencil declarations resolve correctly, and then we write that to disk as `${outputDir}/cli/index.d.ts`. Why is this done in such a convoluted and strange way? It is very brittle and rather confusing I think.

Anyhow, for my change in order for the build to work I need to bundle and copy over `build/cli/config-flags.js` and `build/cli/config-flags.d.ts`. I made changes to do that, but just wanted to flag this since a) the overall way this aspect of building stencil is handled is strange to me and b) I'm not 100% that the way I did this is the best way to do so.